src/base/trafficGraph.js

```diff
@@ -371,6 +374,10 @@ class TrafficGraph extends EventEmitter {
   setState (state, force, parentState) {
     let updatedState = false;
     if (state && Object.keys(state).length > 0) {
+      // it's important to update entryNode when we're switching views
+      if (state.entryNode) {
+        this.entryNode = state.entryNode;
+      }
```

Comment:
This is tricky because there are some cases when `state.entryNode` is undefined. It cause issues with ltrTree layout when we're moving from global to region view and ranker has no idea about entryNode so all layout is broken. I have no idea where setState is called without entryNode but I'll try to look into it